editors/vscode/src/wasm_preview.ts

```diff
@@ -1,10 +1,11 @@
 // Copyright © SixtyFPS GmbH <info@slint.dev>
 // SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
 
+// cSpell: ignore winit
```

Comment:
unrelated, this should probably be ignored globally as this is such a common term in slint